src/app/project/directives/learn-more-button.dir.js

```diff
@@ -0,0 +1,25 @@
+angular.module('app.project').directive('learnMoreButton', [
```

Comment:
It seems to me this directive (and its html template) don't need to be separate, because they're only ever going to be used in one place -- I would just insert directly into `datasharing.form.tpl.html`.